app/services/bio_eddie_service.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+class BioEddieService
```

Comment:
Well we have quite different methods. Not sure that we will have any benefits if it will be an instance. Not so many methods.

Reply:
If it will not be an instance, I think it should be a module, not a class. We only need a class when you mean to have instances of it, in this case it's basically just a collection of namespaced methods, so it should probably be a module.